app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
+	
+Here's an example:
+
+[See it on Plunker](http://plnkr.co/edit/VPktVlI0PWMnYdYVsVBx?p=preview)
+
+```html
+<!-- Create a template for the custom element -->
+<dom-module id='custom-element'>
+  <template>
+    <h1>Heading!</h1>
+    <p>We are elements in custom-element's local DOM.</p>
+  </template>
+</dom-module>
+
+<!-- Drop the custom element on the page -->
```

Comment:
This looks like you're putting this all in the index.html file, which is not going to work on non-Chrome browsers unless you have a workaround in there (it's documented here,  https://2-0-staging-dot-polymer-project.appspot.com/2.0/docs/devguide/registering-elements#main-document-definitions, but as it turns out that's going to need to be updated slightly for 2.0).

The best way to deal with this (and the reason we tend to use plunker) is so we can define custom elements in their own files (`custom-element.html`), and import and use them in another file (`index.html`).

Also, these plunkers need to have the polyfills added so they work on non-Chrome browsers.
Put these together, and this plunker would look like this:

https://plnkr.co/edit/dnL9SeRFZr0sQt0pqKaG?p=preview